rest-api-spec/src/main/resources/rest-api-spec/api/remote_store.stats.json

```diff
@@ -0,0 +1,48 @@
+{
+  "remote_store.stats":{
+    "documentation":{
+      "url": "https://opensearch.org/docs/latest/opensearch/rest-api/remote-store#stats",
```

Comment:
should we add this url after the page exists?

Reply:
removed for now